docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -10,6 +10,27 @@ The plugin provides functionality to test mobile application on real devices, An
 implementation(group: 'org.vividus', name: 'vividus-plugin-mobile-app', version: '{page-component-version}')
 ----
 
+
+include::partial$locator-header.adoc[]
+
+=== Types
+
+[cols="1,1,3", options="header"]
+|===
+
+|Locator type
+|Description
+|Examples
+
+|XPath
+|https://developer.mozilla.org/en-US/docs/Web/XPath[Xpath to locate an element]
```

Comment:
won't it be confusing a link to a web-based locator examples for mobile apps?

Reply:
unfortunately there is no comprehensive mobile-related resources that explains functions, axes and what the xpath, imho xpath on web is equivalent to xpath on mobile because we work with xml, but potentially some functions may not work

Reply:
And I think reusable locators' descriptions should be consistent.

Reply:
the last comment is fixed